nix/crane.nix

```diff
@@ -8,7 +8,13 @@
 let
   common-crane-args = {
     pname = "genealogos";
-    src = crane-lib.cleanCargoSource (crane-lib.path ../.);
+
+    # We need to also include the frontend .html file for the include_str macro in the api
+    src = pkgs.lib.cleanSourceWith {
+      src = crane-lib.path ../.;
+      filter = path: type: (crane-lib.filterCargoSources path type) || (builtins.match ".*html" path != null);
```

Comment:
I feel like the match is a bit loose. What if we end up generating rustdoc, or for any other reason there's a whole lot of html files ending up being checked in? Should we try to filter the exact path instead?